scipy/stats/stats.py

```diff
@@ -6655,18 +6655,19 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     See Also
     --------
     scipy.stats.power_divergence
-    scipy.stats.fisher_exact : A more powerful alternative to the chisquare
-                               test if any of the frequencies are less than 5.
+    scipy.stats.fisher_exact 
+	barnard_exact: An unconditional exact test. Less conservative in small 
+		samples compared to Fisher's Exact test.
 
     Notes
     -----
     This test is invalid when the observed or expected frequencies in each
     category are too small.  A typical rule is that all of the observed
-    and expected frequencies should be at least 5. If one or more frequencies
-    are less than 5, Fisher's Exact Test can be used with greater statistical
-    power. According to [3]_, the total number of samples is recommended to be
-    greater than 13, otherwise a table-based method of obtaining p-values is 
-    recommended.
+    and expected frequencies should be at least 5. When dealing with small 
+	sample sizes, exact tests (such as Fisher's Exact test) are recommended
+	because they do not overreject. According to [3]_, the total number of 
+	samples is recommended to be greater than 13, otherwise an exact test 
+	should be used.
```

Comment:
```suggestion
    and expected frequencies should be at least 5. According to [3]_, the
    total number of samples is recommended to be greater than 13,
    otherwise exact tests (such as Barnard's Exact test) should be used
    because they do not overreject.
```

I think this is a better paraphrase. Avoids writing "exact test should be used" twice (once in the "When dealing ..." statement and again in "According to [3]_ ..." statement.). Also, as Bernard's exact test is often more powerful, it would be better to give that as an example instead of Fisher's exact test.

Reply:
@josef-pkt Do you agree with how this looks now?